lib/ansible/modules/windows/win_get_url.py

```diff
@@ -73,6 +73,40 @@
     type: bool
     default: yes
     version_added: '2.4'
+  checksum:
+    description:
+      - 'If a checksum is passed to this parameter, the digest of the
+        destination file will be calculated after it is downloaded to ensure
+        its integrity and verify that the transfer completed successfully.
+        Format: <checksum>, e.g. checksum="D98291AC[...]B6DC7B97"'
```

Comment:
Please remove this line, for various reasons.
- We use examples to showcase possible values
- It shows the old key=value syntax

If you remove this, you also do not need to quote the whole paragraph.

Reply:
Fixed